frontend/src/metabase/admin/people/containers/NewUserModal.tsx

```diff
@@ -25,7 +25,7 @@ export const NewUserModal = ({ onClose }: NewUserModalProps) => {
   };
 
   return (
-    <ModalContent title="Create user" onClose={onClose}>
+    <ModalContent title={t`Create user`} onClose={onClose}>
```

Comment:
This isn't the file I'm commenting on, but it's closest to the missing change file. We missed 1 translation here

https://github.com/metabase/metabase/blob/5a033232032d0f07f9056f21b06338dca132ddd7/frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx#L92